crates/web5_cli/src/vcs.rs

```diff
@@ -92,6 +121,24 @@ impl Commands {
                         }
                     },
                 };
+                let issuance_date = match issuance_date {
+                    None => None,
```

Comment:
Here it should do the issuance date now()?

Reply:
The SDK will take care of that. This is just meant to be a passthrough from the CLI to the SDK function.